test_config.yaml

```diff
@@ -40,3 +40,23 @@ notifications:
   enabled: True
 
 testing: true
+
+services:
+  paths:
+    - ./baselayer/services
+    - ./skyportal/services
+
+test_server:
+  port: 64502
+  # Reset the cache every * days
+  refresh_cache_days: 7
+  redirects:
+    # List of routes listened for by the test server, and the real URLs that
+    # requests should be redirected to. Requests will be forwarded by the
+    # test server, cached, and responses played back to the API as if it came
+    # from the real server.
+    /node_agent2/node_agent: http://161.72.57.3:8080 # LT
```

Comment:
What is `/node_agent2/node_agent`?

Reply:
This is the API route for LT requests. The full real LT URL is `http://161.72.57.3:8080/node_agent2/node_agent`, but they're not being sent to `http://localhost:cfg["test_server.port"]/node_agent2/node_agent` and this configuration tells the test server that all requests coming in on the `/node_agent2/node_agent` route should be forwarded to the LT server.

To be honest, this is the one part of this setup that I'm a little iffy about. I need a way distinguish which requests want to go to which external system, and the real addresses for those external systems. The method I've settled on is to map API routes to real host address in the configs like this, but I can imagine this raising issues in the future if two external services both have some common route like `"GET /users"` that we want to test and it'll be hard to tell which real APIs to forward the requests to when they're both just being sent to `http://localhost:cfg["test_server.port"]/users` by SP